cmake/modules/Findjsoncpp.cmake

```diff
@@ -0,0 +1,51 @@
+# If not stated otherwise in this file or this component's license file the
```

Comment:
Do not think we need to include this. Can be taken out of the equation as the current JSON processing is only using the Thunder JSON processing.